pkg/types/baremetal/validation/platform.go

```diff
@@ -4,12 +4,16 @@ import (
 	"fmt"
 	"net"
 	"net/url"
+	"reflect"
+
+	"strings"
 
 	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/baremetal"
 	"github.com/openshift/installer/pkg/validate"
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	"strings"
+
+	"github.com/go-playground/validator/v10"
```

Comment:
nit:

please make sure the imports are
```
<stdlib>

<external>

<installer repo>
```

Reply:
Ok thanks I think VC reordered them automatically, I'll fix it